azkaban-web-server/src/web/js/azkaban/view/schedule-panel.js

```diff
@@ -56,6 +56,16 @@ azkaban.SchedulePanelView = Backbone.View.extend({
 
     console.log("current Time = " + scheduleDate + "  " + scheduleTime );
     console.log("cronExpression = " +  scheduleData.cronExpression);
+    var retSignal = validateQuartzStr(scheduleData.cronExpression);
+
+    if(retSignal == 1) {
+      messageDialogView.show("Cron Syntax Error", "A valid Quartz Cron Expression should have 6 or 7 fields.");
+      return;
+    } else if (retSignal == 2) {
```

Comment:
Instead of 1 and 2, consider making them constants. And document the return values in the method signature?